website/content/docs/install-boundary/configure-workers.mdx

```diff
@@ -487,12 +489,11 @@ Complete the following steps to adopt the worker using the CLI:
 
 1. Use the following command to ensure that the `BOUNDARY_ADDR` is set as an environment variable:
 
-    ```hcl
-    $ export BOUNDARY_ADDR="https://c3a7a20a-f663-40f3-a8e3-1b2f69b36254.boundary.hashicorp.cloud"
-    ```
+  ```hcl
+  $ export BOUNDARY_ADDR="https://c3a7a20a-f663-40f3-a8e3-1b2f69b3625  boundary.hashicorp.cloud"
```

Comment:
This looks wrong, should the full stop be added back?
```suggestion
  $ export BOUNDARY_ADDR="https://c3a7a20a-f663-40f3-a8e3-1b2f69b3625.boundary.hashicorp.cloud"
```

Reply:
good catch! not sure how I deleted that